corehq/apps/export/models/new.py

```diff
@@ -58,7 +58,6 @@
     OpenSubCaseAction,
     RemoteApp,
 )
-from corehq.apps.data_dictionary.util import get_deprecated_fields
```

Comment:
I think this was removed maybe in a bad merge?  At any case it looks like it's supposed to be here.  I see a few other import statements were removed.

Reply:
It was duplicate import. There were few more like this that are removed as part of linting.

https://github.com/dimagi/commcare-hq/pull/33158/files#diff-1ae97543378912bde3f3b8e1fc084780195a16a82ea8679a62ffc636e83f9cecR119

I guess removing the one defined later would have made more sense.

Reply:
Update: I'm not sure what happened but after taking the update from remote in local, import was missing. Have added it back. Thanks for pointing it out.